snowplow_tracker/snowplow.py

```diff
@@ -0,0 +1,144 @@
+# """
+#     snowplow.py
+
+#     Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
+
+#     This program is licensed to you under the Apache License Version 2.0,
+#     and you may not use this file except in compliance with the Apache License
+#     Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+#     http://www.apache.org/licenses/LICENSE-2.0.
+
+#     Unless required by applicable law or agreed to in writing,
+#     software distributed under the Apache License Version 2.0 is distributed on
+#     an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#     express or implied. See the Apache License Version 2.0 for the specific
+#     language governing permissions and limitations there under.
+
+#     Authors: Jack Keene, Anuj More, Alex Dean, Fred Blundun, Paul Boocock
+#     Copyright: Copyright (c) 2013-2022 Snowplow Analytics Ltd
+#     License: Apache License Version 2.0
+# """
+import logging
+from typing import  Optional
+from snowplow_tracker import Tracker, Emitter, subject, EmitterConfiguration, TrackerConfiguration
+from snowplow_tracker.typing import Method
+
+# Logging
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+    
+
+"""
+Snowplow Class
+"""
+class Snowplow:
+    _trackers = {}
+
+    @staticmethod
+    def create_tracker( 
+                namespace: str,
+                endpoint: str, 
+                method: Method = "post",
+                emitter_config: EmitterConfiguration = EmitterConfiguration(), 
+                app_id = None,
+                subject: Optional[subject.Subject] = None,
+                tracker_config: TrackerConfiguration = TrackerConfiguration()
+            ) -> Tracker:
+        """
+            Create a Snowplow tracker with a namespace and collector URL
+
+            :param  namespace:          Name of the tracker
+            :type   namespace:          String
+            :param  endpoint:           The collector URL
+            :type   endpoint:           String | None
+            :param  method:             The HTTP request method. Defaults to post.
+            :type   method:             method
+            :param  emitter_config:     Emitter configuration
+            :type   emitter_config:     EmitterConfiguration | None
```

Comment:
```suggestion
            :type   emitter_config:     EmitterConfiguration
```

Also the comment is missing `tracker_config`